sqlglot/expressions.py

```diff
@@ -6542,6 +6542,27 @@ def func(name: str, *args, dialect: DialectType = None, **kwargs) -> Func:
     return function
 
 
+def case(
```

Comment:
Not actually used here - I added it when transpiling NULLIF to a CASE statement. But then I couldn't find any dialects that don't support NULLIf.

Reply:
it's fine